### PR TITLE
[WIP] - Add Getting Started for Docker

### PR DESCRIPTION
**Pull request by sdwheeler**
- Editorial pass on existing install articles
- Added article for how to get started using AzPS in our Docker containers

### CODE REVIEW

File: docs-conceptual/azps-3.5.0/azureps-in-docker.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Docker without `sudo` or local administrative rights. Please follow Docker's official
+[instructions][install] to install `docker` correctly.
+
+The release containers are built from the official PowerShell containers, then the Az module added
+as a layer.
+
+The latest stable image includes:
+
+- PowerShell version 6.2.4
+- Azure PowerShell Az modules version 3.5.0
```
Comment (dcaro) [inline docs-conceptual/azps-3.5.0/azureps-in-docker.md]: We will update the image at each build of the Az module hence the following suggestion.

```suggestion
- Azure PowerShell most current Az module 
```

File: docs-conceptual/azps-3.5.0/azureps-in-docker.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+We are publishing Docker images with Azure PowerShell preinstalled. This article shows you how to
+get started using Azure PowerShell in the Docker container.
+
+## Finding available images
+
+The released images require Docker 17.05 or newer. It is also expected that you are able to run
+Docker without `sudo` or local administrative rights. Please follow Docker's official
+[instructions][install] to install `docker` correctly.
+
+The release containers are built from the official PowerShell containers, then the Az module added
```
Comment (dcaro) [inline docs-conceptual/azps-3.5.0/azureps-in-docker.md]: Is this a typo?
```suggestion
The released containers are built from the official PowerShell containers, then the Az module added
```

File: docs-conceptual/azps-3.5.0/azureps-in-docker.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+- Azure PowerShell Az modules version 3.5.0
+
+A full list of available images can be found on our [Docker image][az image] page.
+
+## Using Azure PowerShell in a container
+
+The following steps show the Docker commands required to download the image and start an interactive
+PowerShell session.
+
+1. Download the latest azure-powershell image.
```
Comment (dcaro) [inline docs-conceptual/azps-3.5.0/azureps-in-docker.md]: The docker run command will pull the latest image if it is not already locally available. Aiming for simplicity, could we remove the download step?

File: docs-conceptual/azps-3.5.0/install-az-ps-msi.md [hunk 4]
```diff
@@ -1,44 +1,42 @@
...
+an offline installer is needed. The recommended way to install Azure PowerShell is with
+PowerShellGet. For instructions on using PowerShellGet to install Azure PowerShell, see
+[Install Azure PowerShell with PowerShellGet](install-az-ps.md).
 
 ## Requirements
 
-The MSI installer for Azure PowerShell works __only__ for PowerShell 5.1 on Windows. For installation on
-non-Windows platforms or later versions of powershell, [Install with PowerShellGet](install-az-ps.md).
-To check your PowerShell version, run the command:
+The MSI installer on Windows and is designed to install Azure PowerShell for PowerShell 5.1 only.
```
Comment (dcaro) [inline docs-conceptual/azps-3.5.0/install-az-ps-msi.md]: I think that there is a typo here.
```suggestion
The MSI installer on Windows is designed to install Azure PowerShell for PowerShell 5.1 only.
```

File: docs-conceptual/azps-3.5.0/install-az-ps.md [hunk 5]
```diff
@@ -130,54 +142,67 @@ Connect-AzAccount
...
 
-You can install or load a specific version of the `Az` module by using the `-RequiredVersion` argument:
+If you have more than one version of the module installed, module autoload and `Import-Module` load
+the latest version by default.
+
+You can install or load a specific version of the `Az` module by using the `-RequiredVersion`
+parameter:
 
 ```powershell-interactive
 # Install Az version 0.7.0
```
Comment (dcaro) [inline docs-conceptual/azps-3.5.0/install-az-ps.md]: Should we update this to version 3.5.0?

Comment (sdwheeler) [reply docs-conceptual/azps-3.5.0/install-az-ps.md]: Yes, or get rid of the comment or change the comment to "latest"

File: docs-conceptual/azps-3.5.0/azureps-in-docker.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+```
+
+## Next steps
+
+To learn more about the Azure PowerShell modules and their features, see
+[Get Started with Azure PowerShell](get-started-azureps.md).
+
+<!-- link references -->
+[install]: https://docs.docker.com/engine/installation/
+[powershell image]: https://hub.docker.com/_/microsoft-powershell
```
Comment (dcaro) [inline docs-conceptual/azps-3.5.0/azureps-in-docker.md]: https://hub.docker.com/_/microsoft-azure-powershell

File: docs-conceptual/azps-3.5.0/azureps-in-docker.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+[az image]: https://hub.docker.com/_/azure-powershell
```
Comment (dcaro) [inline docs-conceptual/azps-3.5.0/azureps-in-docker.md]: https://hub.docker.com/_/microsoft-azure-powershell

**Pull request closed without merging**